### PR TITLE
deprecate testing mode and update README to reflect removal

### PR DESCRIPTION
**Pull request by lanamorgan**
Addresses #31

### CODE REVIEW

**Comment by grantv9** [comment]
Just reviewed and this looks good to me

**Review by grantv9 — APPROVED** [review]

File: isdi [hunk 1]
```diff
@@ -15,12 +15,11 @@ if __name__ == "__main__":
...
-    print("TEST={}".format(config.TEST))
-    init_db(app, sa, force=config.TEST)
+    init_db(app, sa, force=False)
     handler = handlers.RotatingFileHandler('logs/app.log', maxBytes=100000, 
                                            backupCount=30)
     logger = logging.getLogger(__name__)
     logger.setLevel(logging.ERROR)
     logger.addHandler(handler)
-    PORT = 5000 if not config.TEST else 5002
+    PORT = 5000
```
Comment (SantiagoTorres) [inline isdi:24]: Is this change needed to solve #32? I think overall keeping configurability may be desirable.

File: isdi [hunk 2]
```diff
@@ -15,12 +15,11 @@ if __name__ == "__main__":
         print("Checking mode = {}\nApp flags: {}\nSQL_DB: {}"
               .format(config.TEST, config.APP_FLAGS_FILE,
                       config.SQL_DB_PATH))
-    print("TEST={}".format(config.TEST))
-    init_db(app, sa, force=config.TEST)
+    init_db(app, sa, force=False)
```
Comment (SantiagoTorres) [inline isdi:18]: Similarly to below, I wonder if instead we just want to use a ternary to default TEST to false if it does not exist.

**Review by SantiagoTorres — COMMENTED** [review]
Overall this makes sense, I wonder if this change would be more seamless if we just changed config.py:22 to default to "0" so that test is disabled by default...

File: isdi (hunk 2 above)
Comment (lanamorgan) [reply isdi:18]: I thought about this, i.e. should we just change the default or remove it entirely? 

I went with the latter mostly because we're planning on doing quite a bit of work related to  (1) setting up a new testing framework (2) re-factoring the shell scripts into Python scripts and (3) containerizing individual runs and (4) possibly removing the database. 

i think in light of these planned changes it makes more sense to just remove the test mode hooks. but if there's significant disagreement on this, we can just change the defaults.

Comment (SantiagoTorres) [reply isdi:18]: I'm not so too familiar with things, but I wonder if slowly phasing things out will make it less likely to break in existing deployments...

Otherwise things LGTM!

Comment (rchatterjee) [reply isdi:18]: Okay, then let's keep the config.TEST, but change `TEST=0` by default.
